app/ide-desktop/eslint.config.js

```diff
@@ -87,10 +87,6 @@ const RESTRICTED_SYNTAXES = [
         selector: `:matches(ImportDefaultSpecifier[local.name=/^${NAME}/i], ImportNamespaceSpecifier > Identifier[name=/^${NAME}/i])`,
         message: `Don't prefix modules with \`${NAME}\``,
     },
-    {
```

Comment:
Removed it because creating an interface sometimes it too annoying

Reply:
see review comments